pkg/controller/aws/network/vpc/controller_test.go

```diff
@@ -233,12 +233,19 @@ func Test_Create(t *testing.T) {
 				CIDRBlock: "arbitrary cidr block string",
 			},
 		},
+		// Status: v1alpha2.VPCStatus{
```

Comment:
can this just be deleted instead of being left commented out?

Reply:
good catch!